test/runtests.jl

```diff
@@ -111,12 +114,12 @@ end
     @test_throws ArgumentError ZlibDeflateOutputStream(UInt8[], level=10)
     @test_throws ArgumentError ZlibInflateOutputStream(UInt8[], bufsize=0)
 
-    deflated = read(ZlibDeflateInputStream("foo".data))
+    deflated = read(ZlibDeflateInputStream(Vector{UInt8}("foo")))
     buf = IOBuffer()
     out = ZlibInflateOutputStream(buf)
     BufferedStreams.writebytes(out, deflated, length(deflated), true)
     flush(out)
-    @test takebuf_string(buf) == "foo"
+    @test String(take!((buf))) == "foo"
```

Comment:
Viral's wrong. Not being correct about this means anyone who's pinning package versions, which we happen to do even on Compat in cases where reproducibility is important, will get broken packages any time you don't get the dependency right. Check blame on the Compat readme, it's not hard.